cinema/views.py

```diff
@@ -0,0 +1,39 @@
+from django.http import JsonResponse
+from django.shortcuts import render, get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework import status
+from rest_framework.response import Response
+from cinema.models import Movie
+from cinema.serialalizers import MovieSerializer
+
+
+@api_view(['GET', 'POST'])
+def movie_list(request):
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=200)
+    elif request.method == 'POST':
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            return Response(data=serializer.data, status=201)
+        else:
+            return Response(serializer.errors, status=400)
```

Comment:
you don't need it because  `serializer.is_valid(raise_exception=True)` automatically responces with status 400 if serializer is not valid

Reply:
do it